### ISSUE TITLE
name vs. accessible name

### ISSUE THREAD

**Issue by devarshipant** [w3c/wcag21#902]
https://w3c.github.io/wcag21/guidelines/#non-text-content

First bullet "Controls, Input" states "If non-text content is a control or accepts user input, then it has a **name** that describes its purpose..."

Is the control's name its accessible name?

**Comment by awkawk** [comment]
Yes, it is. This is the accessible name. The link to the glossary definition of name (https://w3c.github.io/wcag21/guidelines/#dfn-name) clarifies that this isn't the name attribute in HTML but is the text used to identify the content to the user.
